### ISSUE TITLE
How to set the width of the slides?

### ISSUE THREAD

**Issue by hamza-lahlou** [zulucoda/react-swift-slider#25]
I used react-swift-slider as it mentioned on the documentation but one issue which it takes a 100% of the width and I cannot find any place in the documentation that mention how to override this behavior.

**Comment by zulucoda** [comment]
@hamza-lahlou, hope you are well mate :+1: 

Thank you for reaching out to me, apologies for the late reply. You're correct, the width is set to 100%, and there's currently no CSS handle you could use to override this behaviour.

These are good suggestions man, I will add them into the next version. I'm also thinking of updating the existing height prop to take a string, so you pass in more advanced CSS. I will also do this for width as well. Therefore you can do things like:

```CSS
   width: calc(100% - 80px);
```

I will advise once the new version is done.
Thanks :+1:

**Comment by zulucoda** [comment]
> I used react-swift-slider as it mentioned on the documentation but one issue which it takes a 100% of the width and I cannot find any place in the documentation that mention how to override this behavior.


Hi @hamza-lahlou 

I have published [v7.0.0](https://www.npmjs.com/package/react-swift-slider) which includes the following CSS overrides:

| Class | Description |
| :------| :-----------|
| `.swift-slider-container`   | Override slider container (slider wrapper) |
| `.swift-slider-slides`  | Override slides container |
| `.swift-slider-active-slide`   | Override active slide (current active slide displayed) |
| `.swift-slider-slide`   | Override all slides  |
| `.swift-slider-dots`  | Override dots (bullets) container  |
| `.swift-slider-active-dot`  | Override active dot (bullet) |
| `.swift-slider-dot`  | Override all dots (bullets) |
| `.swift-slider-prev-btn`  | Override previous button |
| `.swift-slider-next-btn`  | Override next button |


See example override below:
```css
.swift-slider-container {
    margin: 0;
    padding: 0;
    border: 5px dotted #C21F39;
    width: 300px;
}
```

produces this:
![Screenshot_2021-02-07 react-swift-slider example](https://user-images.githubusercontent.com/994819/107154511-f8bd9500-697b-11eb-9e3c-ad29c7e925d3.png)

thanks again :+1: 

cheers
muzi
